models/oneinch/ethereum/oneinch_v1_ethereum_trades.sql

```diff
@@ -30,218 +31,218 @@
     ,'contract_address'
 ] %}
 
-WITH oneinch_calls AS
-(
+WITH
+  oneinch_calls AS (
     SELECT
-        call_block_number as block_number,
-        CAST(NULL as string) as taker,
-        fromToken as from_token,
-        toToken as to_token,
-        tokensAmount as from_amount,
-        minTokensAmount as to_amount,
-        call_tx_hash as tx_hash,
-        call_block_time as block_time,
-        call_trace_address as trace_address,
-        CAST(-1 as integer) AS evt_index,
-        contract_address
+      call_block_number AS block_number,
+      TRY_CAST(NULL AS VARCHAR) AS taker,
+      fromToken AS from_token,
+      toToken AS to_token,
+      tokensAmount AS from_amount,
+      minTokensAmount AS to_amount,
+      call_tx_hash AS tx_hash,
+      call_block_time AS block_time,
+      call_trace_address AS trace_address,
+      TRY_CAST(-1 AS INTEGER) AS evt_index,
+      contract_address
     FROM
         {{ source('oneinch_ethereum', 'exchange_v1_call_aggregate') }}
     WHERE
         call_success
         {% if is_incremental() %}
         AND call_block_time >= date_trunc("day", now() - interval '1 week')
```

Comment:
i believe this should fail, we had to reformat the interval syntax

Reply:
when you ran `dbt compile`, it pulled the `non-incremental` jinja logic since you haven't built the spell locally yet (or at the time), but if you ran the spell, then compiled, it would spit out the incrmental logic. this is likely why you've missed it here.

Reply:
You're right! Good catch! Also just realized that trino apparently doesn't allow `- interval '1' week` so will re-define this in days